### PR TITLE
Add the ANL MPICH Compatibility white paper

### PR DESCRIPTION
**Pull request by SteveO-Cray**
New version of the Shasta MPICH ABI compatibility white paper.  First version to be put on github.

Any suggestions/comments/corrections are welcome.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ related DPM functionality, as well as a Cray extension, _MPIX\_Comm\_rankpool_.
 
 [Download MPI Dynamic Process Management Whitepaper](https://github.com/PE-Cray/whitepapers/raw/master/mpt/mpi-spawn.doc)
 
+
+**MPICH ABI Compatibility Status for HPE Cray EX Systems**
+
+<p align="justify">
+This document describes ANL MPICH ABI Compatibility on HPE Cray EX systems and how use this capability.
```
Comment (naveen-rn) [inline index.md]: how use this capability -> how to use this capability

**Review by naveen-rn — APPROVED** [review]
Please fix this and merge the changes

File: index.md (hunk 1 above)
Comment (SteveO-Cray) [reply index.md]: Fixed.

**Comment by SteveO-Cray** [comment]
FYI - cannot merge.  I have an error:
```
Merging is blocked

The base branch requires all commits to be signed.
```
Was this intentional?  I started looking at the instructions, I think I can figure this out.

**Comment by SteveO-Cray** [comment]
Naveen - I installed GPG on my macbook, then amended the commit in my pull request.  I made the one change you requested and used "-S" to sign the commit while amending. I then force pushed the development, and github no longer complains about unsigned commits.  I hope this is what I was suppose to do.

Please re-review to confirm that I have done everything correct.  I will not merge until you re-review.

**Comment by naveen-rn** [comment]
Looks ok to me. Merging.

**Comment by SteveO-Cray** [comment]
Thanks!

**Pull request merged**